Packs/IBMResilientSystems/Integrations/IBMResilientSystems/README.md

```diff
@@ -861,3 +864,364 @@ Required:true
 Status:Closed
 }
 ]</pre></div></div></section>
+<hr>
+<h3 id="h_9184065563181528813233520">Add a note to an incident: rs-add-note</h3>
+<p>Add a note to an incident.</p>
+<h5>Command Example</h5>
+<p><code>!rs-add-note incident-id=2111 note="This is a note"</code></p>
+<h5>Input</h5>
+<table style="height: 287px; width: 737px;" border="2" cellpadding="6">
+<tbody>
+<tr>
+<td style="width: 211px;"><strong>&nbsp;Parameter</strong></td>
+<td style="width: 503px;"><strong>Description</strong></td>
+</tr>
+<tr>
+<td style="width: 211px;">incident-id</td>
+<td style="width: 503px;">
+<p>Incident ID to add the note there</p>
```

Comment:
Are we able to move this to Markdown? It would make maintenance in the future much easier and help to reduce LoC @reutshal